### ISSUE TITLE
Allow pre-release versions of marshmallow 3 in setup.py?

### ISSUE THREAD

**Issue by maxalbert** [timakro/apispec-oneofschema#2]
The `setup.py` file specifies `'marshmallow<3.0.0'` as a requirement, but the latest version seems to work fine with the pre-releases of marshmallow (at least in the few cases that I have tried).

The problem is that if I try to install `apispec-oneofschema` using pipenv, it will uninstall the latest version of marshmallow and install `v2.18.1` instead because of this requirement.

Is it possible to tweak this line in `setup.py` so that the pre-release versions of marshmallow do not conflict with this package? Or is there anything in `apispec-oneofschema` that currently prevents running with those pre-release versions of marshmallow?

Many thanks!

**Comment by timakro** [comment]
You're absolutely right. Seems what always made me think it's not working is a problem with the 1.x versions of marshmallow-apispec. They don't work with marshmallow 3 but don't have this restriction in the dependencies.

I released version 2.0.1 and updated the dependencies like you suggested.

So to work around the problem with marshmallow-apispec's dependencies you need to install the pre-release 2.x versions of the package explicitly as well.

```
pip install --pre marshmallow
pip install --pre marshmallow-oneofschema
pip install apispec-oneofschema
```

**Comment by maxalbert** [comment]
Awesome, thank you for the quick response and the new release! 👍
